src/views/AssessmentCell/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { ChevronLeftLineIcon } from '@ifrc-go/icons';
+import Container from '#components/Container';
+import IconButton from '#components/IconButton';
+import Image from '#components/Image';
+import TextOutput from '#components/TextOutput';
+import useTranslation from '#hooks/useTranslation';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const strings = useTranslation(i18n);
+
+    const navigate = useNavigate();
+
+    const handleBackButtonClick = useCallback(() => {
+        navigate(-1);
+    }, [navigate]);
```

Comment:
@frozenhelium What should be the behavior of the back button? Should we just go back on history

Reply:
Not sure, we need to check how it was implemented previously